MpsAllocatorSample/Program.cs

```diff
@@ -0,0 +1,212 @@
+﻿using PlayFab;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace MpsAllocator
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            Console.WriteLine("Welcome to the MpsAllocatorSample!\nThis sample allows you to easily call frequently used APIs on your MPS Build");
+            Console.WriteLine("Enter TitleID");
+            PlayFabSettings.staticSettings.TitleId = Console.ReadLine();
+            
+            Console.WriteLine("Enter developer secret key");
+            PlayFabSettings.staticSettings.DeveloperSecretKey = Console.ReadLine();
+
+            var req = new PlayFab.AuthenticationModels.GetEntityTokenRequest();
+
+            var res = await  PlayFabAuthenticationAPI.GetEntityTokenAsync(req);
+
+            while (true)
+            {
+                int option = PrintOptions();
+
+                switch (option)
+                {
+                    case 1:
+                        await RequestMultiplayerServer();
+                        break;
+                    case 2:
+                        await ListBuildSummaries();
+                        break;
+                    case 3:
+                        await GetBuild();
+                        break;
+                    case 4: 
+                        await ListMultiplayerServers();
+                        break;
+                    case 5: 
+                        await ListVirtualMachineSummaries();
+                        break;
+                    default:
+                        Console.WriteLine("Please enter a valid option");
+                        continue;
+                }
+            }
+
+        }
+
+        static int PrintOptions()
+        {
+            string errorMsg = "Please enter a valid option (1-5)";
+            while (true)
```

Comment:
nit: Same this here, user would have to enter control+c, which is probably ok, but if we can have q or something to signal an end that would be friendlier.

Reply:
that would require more validation on the callers though, so I'd rather leave it aside for the time being, since the purpose of this sample is to show devs how easy it is to do allocation calls on our platform